net/netdev/netdev_ioctl.c

```diff
@@ -636,235 +636,204 @@ static FAR struct net_driver_s *netdev_ifr_dev(FAR struct ifreq *req)
 static int netdev_ifr_ioctl(FAR struct socket *psock, int cmd,
                             FAR struct ifreq *req)
 {
-  FAR struct net_driver_s *dev;
-  int ret = -EINVAL;
+  FAR struct net_driver_s *dev = NULL;
+  FAR struct net_driver_s *tmpdev;
+  ssize_t arglen;
+  int ret;
 
   ninfo("cmd: %d\n", cmd);
 
   net_lock();
 
-  /* Execute the command */
+  /* Execute the command without ifr_name or lifr_name */
```

Comment:
Maybe change comment to:

`/* Execute commands that do not need ifr_name or lifr_name */`

or something like that?

Reply:
Thank you, done!